.github/workflows/ci.yaml

```diff
@@ -75,7 +80,7 @@ jobs:
         id: cache
         with:
           path: ${{ matrix.prefix }}
-          key: ${{ matrix.label }}-conda-${{ hashFiles('envs/environment.yaml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
+          key: ${{ matrix.label }}-conda-${{ hashFiles('envs/environment.yaml') }}-${{ env.DATE }}-${{ env.CONDA_CACHE_NUMBER }}
 
       - name: Update environment due to outdated or unavailable cache
```

Comment:
Instead of trying to refresh the cache for databundle / cutouts, maybe just delete the two folders if BUNDLE_CACHE_NUMBER changed (or maybe do it differently: Name the variable REFRESH_DATA_CACHE and delete the folder if == 1; keep folders if ==0).

Reply:
NVM. Resolved with the same comment as above.